optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -114,8 +116,8 @@ def run(command, ignore_status=False, quiet=True, timeout=60):
     end_time = time.time()
     duration = end_time - start_time
     result.command = command
-    result.stdout = str(fabric_result)
-    result.stderr = fabric_result.stderr
+    result.stdout = str(fabric_result.stdout)
+    result.stderr = str(fabric_result.stderr)
```

Comment:
Good.  Fabric can be surprisingly different from version to version (just found yet another example while packaging version 46.0).